Watchman.Web.Server/Areas/Messages/Controllers/MessagesController.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Threading.Tasks;
+using Watchman.Cqrs;
+using Watchman.DomainModel.Messages;
+using Watchman.DomainModel.Messages.Queries;
+using Watchman.DomainModel.Responses.Commands;
+using Watchman.DomainModel.Responses.Queries;
+using Watchman.Web.Server.Areas.Commons.Integration;
+using Watchman.Web.Server.Areas.Messages.Models.Dtos;
+using Watchman.Web.Server.Areas.Responses.Models.Dtos;
+
+namespace Watchman.Web.Server.Areas.Messages.Controllers
+{
+    public class MessagesController : BaseApiController
+    {
+        private readonly IQueryBus _queryBus;
+
+        public MessagesController(IQueryBus queryBus)
+        {
+            this._queryBus = queryBus;
+        }
+
+        [HttpGet]
+        public async Task<IEnumerable<MessageDto>> GetMessages(ulong serverId, ulong channelId,  ulong? userId)
```

Comment:
te parametry lepiej zmienić na klase, spójrz jak jest w ChannelsController 

wtedy (według mnie) prościej sie to obsługuje z poziomu frontu
bo masz jeden obiekt do wysłania